bgpd/bgp_route.c

```diff
@@ -3454,7 +3454,8 @@ int bgp_update(struct peer *peer, struct prefix *p, uint32_t addpath_id,
 		    && (safi == SAFI_UNICAST || safi == SAFI_LABELED_UNICAST))
 		    || (safi == SAFI_EVPN &&
 			bgp_evpn_is_prefix_nht_supported(p))) {
-			if (peer->sort == BGP_PEER_EBGP && peer->ttl == 1
+			if (safi != SAFI_EVPN
+			    && peer->sort == BGP_PEER_EBGP && peer->ttl == 1
```

Comment:
can we replace 1 with BGP_DEFAULT_TTL?

Reply:
yes -- no magic numbers... using the #def would be better here

Reply:
We will fix this. But Just to clarify, the ttl part is not that was modified now, but already present in the code.